UpdateAssemblyInfoTask/UpdateAssemblyInfo.ps1

```diff
@@ -7,10 +7,11 @@
     [string]$assemblyCompany,
     [string]$assemblyTrademark,
     [string]$assemblyConfiguration,
-    [string]$assemblyVersion, 
-    [string]$assemblyFileVersion, 
+    [string]$assemblyVersion,
+    [string]$assemblyFileVersion,
     [string]$assemblyInformationalVersion,
-    [string]$customAttributes
+    [string]$customAttributes,
+    [boolen]$addMissingAttributes
```

Comment:
I know this is old as this repo seems to be abandoned, but there is a typo on this line. You can just make this [bool]